flybywiresim/aircraft
# fix(CDU): auto weight import in MCDU uses target values with gsx now

Fixes #[issue_no]

## Summary of Changes


This is a quick fix for the auto import of the ZFW and ZFWCG during boarding in the MCDU.
For our own boarding this function uses the target values of the boarding process. 

This fix makes sure that same functionality extends to GSX boarding as well.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):\_chaoz_

## Testing instructions


1. Start C/D
2. make sure GSX payloadsync is disabled in the 3rd party EFB options.
3. Set some random boarding values or use a Simbrief import
4. Start boarding
5. Set up mcdu enough to be able to import weights on the INIT B page
6. Press the line select key next to the empty ZFW/ZFWCG entry
7. Values that show in the scratchpad should be the target boarding values set in the EFB
8. Repeat the entire process with GSX payload sync enabled and using GSX for boarding



## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
